build.gradle

```diff
@@ -24,23 +24,34 @@ subprojects {
         addons
     }
 
-    // rosuda jars are required for both par-connector and par-script
-    dependencies {
-        addons "org.rosuda:jriengine:0.5.x"
-        addons "org.rosuda:jri:0.5.x"
-        addons "org.rosuda:rengine:1.7.x"
-    }
 }
 
-project(':par-script') {
+def isWindows = System.properties['os.name'].toLowerCase().contains('windows')
+def isMac = System.properties['os.name'].toLowerCase().contains('mac')
+def isLinux = System.properties['os.name'].toLowerCase().contains('linux')
+def rHome = System.getenv('R_HOME')
+
+def rExec
+def rScript
+if (isWindows) {
+    def ARCH = System.getenv('ProgramFiles(x86)') != null ? 'x64' : 'i386'
+    rExec = rHome ? rHome + File.separator + 'bin' + File.separator + ARCH + File.separator + 'R.exe' : 'R.exe'
+    rScript = rHome ? rHome + File.separator + 'bin' + File.separator + ARCH + File.separator + 'Rscript.exe' : 'Rscript.exe'
+} else {
+    rExec = rHome ? rHome + File.separator + 'bin' + File.separator + 'R' : 'R'
+    rScript = rHome ? rHome + File.separator + 'bin' + File.separator + 'Rscript' : 'Rscript'
+}
```

Comment:
Would be nice if there is something that check if R executable is available (installed) and suggests to install it if not.